### ISSUE TITLE
How to run post query given by using path params as well as passing body fields

### ISSUE THREAD

**Issue by sadath-12** [nineinchnick/trino-openapi#110]
The equivalent curl request would be 

```sh
curl -X 'POST' \
  'http://<ip:port>/namespaces/{namespace}/indexes/{index}/search' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{

  "include_content": true,
  "k": 5,
  "query": "my query"
}'
```

```namespace``` and ```index``` is path params provided at every query query

```include_content``` ```k``` ```query``` are body fields 

This is the spec yaml for that particular request 

```yaml
 /namespaces/{namespace}/indexes/{index}/search:
    post:
      operationId: index_search
      parameters:
      - explode: false
        in: path
        name: namespace
        required: true
        schema:
          type: string
        style: simple
      - explode: false
        in: path
        name: index
        required: true
        schema:
          type: string
        style: simple
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/SearchRequest'
        description: ""
        required: true
      responses:
        "200":
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/IndexSearchResponse'
          description: Index search results
        "500":
          description: Unable to search index
      summary: Search a vector index in a namespace
      tags:
      - retrieval
  ````

**Comment by vcknorket** [comment]
Thanks for posting this. We have a similar problem. @nineinchnick Would appreciate any support with this

**Comment by nineinchnick** [comment]
Fixed in 1134a7f0fde09327418baeb2adbfa46ef06d9f1a

This connector can combine multiple endpoints into a single table. It also tries using POST HTTP methods for SELECT queries, if an endpoint doesn't define a GET method. There was a bug in how required path parameters were handled. I added a test with the spec above so we can avoid such issues in future. Thanks for reporting this!
